designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/testing/DeploymentManagerStub.scala

```diff
@@ -61,7 +61,7 @@ class DeploymentManagerStub extends DeploymentManager with AlwaysFreshProcessSta
   override def customActions: List[CustomAction] = Nil
 
   override def invokeCustomAction(actionRequest: CustomActionRequest, canonicalProcess: CanonicalProcess): Future[CustomActionResult] =
-    Future.failed(CustomActionNotImplemented(actionRequest))
+    Future.failed(new IllegalArgumentException("Not implemented"))
```

Comment:
We have a dedicated `NotImplemented` http status. WDYT about throwing scala's `NotImplementedError` by `???` and handling it in `EspErrorToHttp.errorToHttp`?

Reply:
Good point. Fixed.